normandy/schema.py

```diff
@@ -9,4 +9,16 @@ class NormandyQuery(BaseQuery, RecipesQuery, StudiesQuery, graphene.ObjectType):
     pass
 
 
-schema = graphene.Schema(query=NormandyQuery)
+class DisableIntrospectionMiddleware:
+    """
+    This class hides the introspection.
+    """
+
+    def resolve(self, next, root, info, **kwargs):
+
+        if info.field_name.lower() in ['__schema', '_introspection']:
```

Comment:
We are looking for private fields here, where do they come from? A link maybe?

Reply:
I changed up the list, with a list that is listed from https://graphql.org/learn/introspection/